app/controllers/dashboard_controller.rb

```diff
@@ -46,6 +46,11 @@ def edit_booking
     redirect_to root_path(date: booking_params[:booking_date])
   end
 
+  def delete_booking
+    @resource.destroy
```

Comment:
@tochman ok, also not sure what happened but the delete button is now showing when you first create a booking and wasn't showing before it only showed when one clicked to edit booking?